backend/src/main/java/com/benine/backend/http/CameraInfoHandler.java

```diff
@@ -28,11 +27,6 @@ public CameraInfoHandler(Logger logger) {
   public void handle(HttpExchange exchange) throws IOException {
     getLogger().log("Got an http request with uri: "
             + exchange.getRequestURI(), LogEvent.Type.INFO);
-    try {
-      respond(exchange, getCameraController().getCamerasJSON());
-    } catch (CameraConnectionException e) {
-      getLogger().log("Cannot connect to camera with id: " + e.getCamId(), LogEvent.Type.CRITICAL);
-    }
-    respond(exchange, "{\"succes\":false}");
+    respond(exchange, getCameraController().getCamerasJSON());    
```

Comment:
I might be wrong but since getCamerasJSON does not return succes false when it fails to connect the camera it would be unclear to the client if the camera is not existent or if its unreachable.